File: backend/app/modules/fx/manifest.py
# DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
# Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
"""Currency / FX module manifest."""

from app.core.module_loader import ModuleManifest

manifest = ModuleManifest(
    name="oe_fx",
    version="0.1.0",
    display_name="Currency / FX",
    description="Live ECB foreign-exchange rates and optional PPP conversion for multi-currency cost bases and estimates",
    author="OpenConstructionERP Core Team",
    category="core",
    depends=["oe_costs"],
    auto_install=True,
    enabled=True,
)

File: frontend/src/modules/schedule/manifest.ts
// DDC-CWICR-OE: DataDrivenConstruction · OpenConstructionERP
// Copyright (c) 2026 Artem Boiko / DataDrivenConstruction
import { CalendarDays } from 'lucide-react';
import type { ModuleManifest } from '../_types';

// internal-id ddc-lineage:a17f93c4-schedule-02
export const manifest: ModuleManifest = {
  id: 'schedule',
  name: 'schedule.title',
  description: 'modules.schedule_desc',
  version: '1.0.0',
  icon: CalendarDays,
  category: 'planning',
  defaultEnabled: true,
  routes: [],
  navItems: [],
};
